src/main/java/mate/academy/dao/impl/CarDaoImplJdbc.java

```diff
@@ -0,0 +1,206 @@
+package mate.academy.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.CarDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Car;
+import mate.academy.model.Driver;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImplJdbc implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection(); PreparedStatement statement =
+                connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getLong(1));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Something went wrong. Can't create car "
+                    + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT cars.id, cars.model, cars.manufacturer_id, "
+                + "manufacturers.name, manufacturers.country FROM cars "
+                + "JOIN manufacturers ON cars.manufacturer_id = "
+                + "manufacturers.id WHERE cars.id = ? AND cars.deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection(); PreparedStatement statement =
+                connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(parseDataFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Something went wrong. Can't get car by ID "
+                    + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String query = "SELECT cars.id, cars.model, cars.manufacturer_id, manufacturers.name, "
+                + "manufacturers.country FROM cars "
+                + "LEFT JOIN manufacturers ON cars.manufacturer_id = manufacturers.id "
```

Comment:
why left join?

Reply:
It`s just trying to understabd if it works with tables in Left Join. Forgoten to change in this query.  Fixed.